api/agentpb/collector.proto

```diff
@@ -307,12 +307,12 @@ message MetricsBucket {
     float m_temp_blks_written_cnt = 21;
     // Total number of temp blocks written by the statement.
     float m_temp_blks_written_sum = 22;
-    float m_blk_read_time_cnt = 23;
-    // Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
-    float m_blk_read_time_sum = 24;
-    float m_blk_write_time_cnt = 25;
-    // Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
-    float m_blk_write_time_sum = 26;
+    float m_shared_blk_read_time_cnt = 23;
+    // Total time the statement spent reading shared blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
+    float m_shared_blk_read_time_sum = 24;
+    float m_shared_blk_write_time_cnt = 25;
+    // Total time the statement spent writing shared blocks, in milliseconds (if track_io_timing is enabled, otherwise zero).
+    float m_shared_blk_write_time_sum = 26;
```

Comment:
It should be safe to change as agent and server don't use json API for interaction.

Reply:
are we sure? Have you tested it?

Reply:
Yep, checked, it works